DataCapturing/Source/Persistence/V11Database.swift

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright 2023 Cyface GmbH
+ *
+ * This file is part of the Cyface SDK for iOS.
+ *
+ * The Cyface SDK for iOS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface SDK for iOS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface SDK for iOS. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import CoreMotion
+import CoreData
+import OSLog
+
+/**
+ Represents all the data only required by this branch of the SDK. This currently encompasses altitude data and related structures.
+
+ The altitude can be provided either by an altimeter or by the sattelite based geo location sensor (GPS, etc.).
+
+ - Author: Klemens Muthmann
+ */
+public class V11Database {
+    // MARK: - Properties
+    /// A *CoreData* Stack used to
+    private let coreDataStack: CoreDataManager
+    /// The minimum number of meters before the ascend is increased, to filter sensor noise.
+    private static let ascendThresholdMeters = 2.0
+    /// Only altimeter changes above this value are considered to be valid height changes. Everything below is considered to be noise.
+    private static let altimeterNoiseThreshold = 0.1
```

Comment:
This is slightly different from the Android implementation, where we use
- a sliding windows to average (smoothen) the pressure samples
- then use these averages to calculate the altitude from pressure
- and then use the same ascentThreshold = 2 m as for GNSS altitudes

Is there a reason why you use an alpha filter, instead?

I tried this before (instead of the sliding window) but at least in the Android context with the bug (sensitive display on Pixel 6 which influences the pressure measured) the alpha filter did not work better then the siding window, when comparing with the Pixel 3a without this bug.

I.e. I cannot really change this on Android without having to test and compare both devices/barometers again, which took quite some time.

Reply:
Just wondering if there is a reason for this different approach on iOS.
At least it would be good to document this design decision, so we don't think both implementations are equal.

I guess it makes sense to use another approach on iOS as the specific Pixel 6 barometer "behavior" was not yet noticed on iOS.

Reply:
I couldn't find the implementation in our Android Code, so I took the most obvious solution and decided to wait till someone complains about this.